resources/views/vendor/pagination/bootstrap-4.blade.php

```diff
@@ -0,0 +1,44 @@
+@if ($paginator->hasPages())
+    <ul class="pagination" role="navigation">
+        {{-- Previous Page Link --}}
```

Comment:
Comments style are:

`<!-- Previous Page Link - OPEN -->`
code
`<!-- Previous Page Link - CLOSE -->`

This helps to understand what is the code block doing and where it starts and finish.